.jazzy.yml

```diff
@@ -0,0 +1,47 @@
+module: SwiftBSON
+author: Neal Beeken, Nellie Spektor, Patrick Freed, and Kaitlin Mahar
+author_url: https://github.com/mongodb/swift-bson
+github_url: https://github.com/mongodb/swift-bson
+theme: fullwidth
+swift_build_tool: spm
+readme: "etc/docs-main.md"
+root_url: https://mongodb.github.io/swift-bson/docs/SwiftBSON/
+documentation: "Guides/*.md"
+
+custom_categories:
+  - name: General Guides
+    children:
+      - BSON-Guide
```

Comment:
documentation files are lumped in with types, so I needed to rename this file to disambiguate it from the BSON enum.